libs/pika/errors/include/pika/errors/error_code.hpp

```diff
@@ -84,13 +89,14 @@ namespace pika {
         ///
         /// \param mode   The parameter \p mode specifies whether the constructed
         ///               pika::error_code belongs to the error category
-        ///               \a pika_category (if mode is \a plain, this is the
-        ///               default) or to the category \a pika_category_rethrow
-        ///               (if mode is \a rethrow).
+        ///               \a pika_category (if mode is \a throwmode::plain, this
```

Comment:
I guess if we add `throwmode::` in front of `plain` we should also add it in front of `rethrow`, or leave it as it is, I think it's quite clear

Reply:
No I think it's good to update them. Good catch. Since this is documentation it wasn't checked for me... I'll update these.

Reply:
Updated.